src/response.ts

```diff
@@ -50,10 +55,11 @@ const handleFinalStatusResult = (idStatus: GENERIC_STATUS, authorizationCode?: s
 };
 
 const showFinalResult = (outcome: OutcomeEnumType) => {
+  const viewOutcome: string = ViewOutcomeEnumType.decode(outcome).getOrElse(ViewOutcomeEnum.GENERIC_ERROR).toString();
```

Comment:
👍 with this trick on `decode` all unhandled errors should be map in to `GENERIC_ERROR`